### PR TITLE
refactor scenes panel

### PR DESCRIPTION
**Pull request by aditya-mitra**
## Summary

**Old File Structure**

```
.
 |-index.tsx
 |-index.stories.tsx
 |-modals
 | |-RenameScene.tsx
 |-container
 | |-index.tsx
 | |-index.stories.tsx
```

**Current File Structure**

```
.
 |-index.tsx
 |-SceneItem.tsx
 |-RenameSceneModal.tsx
```

### CODE REVIEW

File: packages/editor/src/panels/scenes/SceneItem.tsx [hunk 1]
```diff
@@ -26,37 +26,33 @@ import { PopoverState } from '@ir-engine/client-core/src/common/services/Popover
...
   scene: StaticResourceType
-  moveMenuUp?: boolean
   handleOpenScene: () => void
   refetchProjectsData: () => void
   onRenameScene?: (newName: string) => void
   onDeleteScene?: (scene: StaticResourceType) => void
 }
 
-export const SceneItem = ({
+export default function SceneItem({
```
Comment (hanzlamateen) [inline packages/editor/src/panels/scenes/SceneItem.tsx:49]: Please create a PR for MT, as same control is used in it. And this change is breaking the import in MT.

File: packages/editor/src/panels/scenes/SceneItem.tsx [hunk 2]
```diff
@@ -26,37 +26,33 @@ import { PopoverState } from '@ir-engine/client-core/src/common/services/Popover
...
   scene,
-  moveMenuUp,
```
Comment (hanzlamateen) [inline packages/editor/src/panels/scenes/SceneItem.tsx:54]: moveMenuUp was used to fix some issue regarding context menu getting cut when screen size is small.

https://github.com/EtherealEngine/etherealengine/pull/10934
https://tsu.atlassian.net/browse/IR-3754

**Review by hanzlamateen — DISMISSED** [review]

File: packages/editor/src/panels/scenes/SceneItem.tsx (hunk 1 above)
Comment (aditya-mitra) [reply packages/editor/src/panels/scenes/SceneItem.tsx:49]: yes had made the change in [the MT PR](https://github.com/theinfinitereality/irpro-multitenancy/pull/460/files)

File: packages/editor/src/panels/scenes/SceneItem.tsx (hunk 2 above)
Comment (aditya-mitra) [reply packages/editor/src/panels/scenes/SceneItem.tsx:54]: right. we had changed it to [the `Popup` component](https://github.com/ir-engine/ir-engine/pull/125/files#diff-283abec6259bf451de484772618afba90537674c2460107affcd9b8d5470155aR93) which handles dynamic positioning

**Review by CITIZENDOT — APPROVED** [review]

**Pull request merged**